bbb-convert-stage-5.sh

```diff
@@ -113,26 +113,18 @@ build_grub_efi() {
   fi
 
   mkdir -p $grub_arm
-  mkdir -p $grub_linux
-
-  # First build linux tools.
+  # Build GRUB tools (grub-mkimage) and ARM modules in one step
   ./autogen.sh
-  ./configure --quiet CC=gcc --target=x86_64 --with-platform=efi --prefix=$grub_linux
-  make --quiet
-  make --quiet install
-
-  # Clean workspace.
-  make --quiet clean
-  make --quiet distclean
-
-  # Now build ARM modules.
-  ./configure --host=$toolchain --with-platform=efi --prefix=$grub_arm \
-      CFLAGS="-Os -march=armv7-a" CCASFLAGS="-march=armv7-a" --disable-werror
-  make --quiet
-  make --quiet install
-
-  # Build grub.efi binary.
-  $grub_linux/bin/grub-mkimage  -v -p /$efi_boot -o grub.efi --format=arm-efi \
+  ./configure --host=x86_64-linux-gnu TARGET_CC=${toolchain}"-"gcc \
+     TARGET_OBJCOPY=${toolchain}"-"objcopy \
+     TARGET_STRIP=${toolchain}"-"strip \
+     TARGET_NM=${toolchain}"-"nm \
+     TARGET_RANLIB=${toolchain}"-"ranlib \
+     --target=arm --with-platform=efi --exec-prefix=$grub_arm \
+     --prefix=$grub_arm
+  make -j4
+  make install
+  ./grub-mkimage  -v -p /$efi_boot -o grub.efi --format=arm-efi \
       -d $grub_arm/lib/grub/arm-efi/  boot linux ext2 fat serial part_msdos \
```

Comment:
change $grub_arm/lib/grub/arm-efi/ to $grub_build/lib/grub/arm-efi/

Keep in mind to change a following path in install_files():
sudo install -m 0755 ${output_dir}/grub/arm/bin/grub-editenv $rootfs_dir/usr/bin
to
sudo install -m 0755 ${output_dir}/grub/build/bin/grub-editenv $rootfs_dir/usr/bin